src/components/boilersApp/modal/button/index.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Button.css';
+
+function Button({
+  btnLabel, onClick, primary, boilerId,
+}) {
+  return (
+    <button type="button" className={`Button ${primary ? 'Button-primary' : ''}`} onClick={onClick.bind(this, boilerId)}>
```

Comment:
What is the functionality of the `bind` here? 🤔
I guess you can use an arrow function instead of that bind
```suggestion
    <button type="button" className={`Button ${primary ? 'Button-primary' : ''}`} onClick={() => onClick(boilerId)}>
```